blocks/carousel/__slide-button/_type/carousel__slide-button_type_next.css

```diff
@@ -0,0 +1,11 @@
+.carousel__slide-button_type_next {
+  background-image: url(../../../../images/button-slide-next.png);
```

Comment:
Don't forget to unify the code style. Do you use quotes inside `url()` statement or not?